bazel/README.md

```diff
@@ -46,6 +46,7 @@ for how to update or override dependencies.
        cmake \
        automake \
        autoconf \
+       libc++1 \
```

Comment:
Hmm, which configuration are you running? ASAN/TSAN with libc++? In general system installed libc++ could be out-of-date so I don't like them (it sometime cause hard-to-debug issue). 

Likely you need LD_LIBRARY_PATH to where `llvm-config --libdir` point to, or pass it in `-Wl,rpath,<libdir>`, setup that in bazel/setup_clang.sh`?

Reply:
I was just trying to do a normal build, no special options, and I got this complaint:

```
FAILED: gen/builtins-generated/bytecodes-builtins-list.h
python ../../tools/run.py ./bytecode_builtins_list_generator gen/builtins-generated/bytecodes-builtins-list.h
./bytecode_builtins_list_generator: error while loading shared libraries: libc++.so.1: cannot open shared object file: No such file or directory
```

This was resolved, for me, by installing libc++1. What's the right way to resolve it? The suggestions you gave seem to imply that I already have a libc++1 (which might be the case) but I didn't know where that was.

Reply:
Don't you have `config=libc++` in some of your bazelrc? The right way is point LD_LIBRARY_PATH to where `libc++.so.1` is (which is `llvm-config --libdir`), or you might actually using Google internal toolchain?